tests/worksheets/src/test/scala/tests/worksheets/WorksheetSuite.scala

```diff
@@ -51,6 +51,24 @@ class WorksheetSuite extends BaseSuite {
        |""".stripMargin
   )
 
+  checkDecorations(
```

Comment:
As this is not just a worksheet issue (it can be reproduced by vanilla mdoc), would you mind adding a test to the regular suites?

Reply:
I added the test case to `DefaultSuite.scala`, seems to work alright.